pipeline_steps/common.groovy

```diff
@@ -537,6 +537,18 @@ void clone_external_repo(String directory, String repo, String ref, String refsp
 }
 
 
+void merge_pr_chain(String directory='./', List pullRequestIds=null){
+  println("Attempting to merge the following pull requests onto the base branch: ${pullRequestIds}.")
+  for (pullRequestId in pullRequestIds) {
+    println("Merging pull request ${pullRequestId}.")
+    sh """#!/bin/bash -xe
+      cd ${directory}
```

Comment:
Nit, but can you move the `cd` outside of the loop so it doesn't keep `cd`ing back to the same place?